editor/js/editor-libs/events.js

```diff
@@ -8,14 +8,14 @@ import * as cssEditorUtils from "./css-editor-utils.js";
 function addCSSEditorEventListeners(exampleChoiceList) {
   exampleChoiceList.addEventListener("cut", copyTextOnly);
   exampleChoiceList.addEventListener("copy", copyTextOnly);
-  exampleChoiceList.addEventListener("paste", handlePasteEvents);
 
   exampleChoiceList.addEventListener("keyup", function (event) {
     var exampleChoiceParent = event.target.parentElement;
 
     cssEditorUtils.applyCode(
       exampleChoiceParent.textContent,
-      exampleChoiceParent
+      // keyup events fire on the .cm-scroller four levels down from the choice div
+      exampleChoiceParent.parentElement.parentElement.parentElement
```

Comment:
Any chance we can use [`closest()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) to get the desired element?

Reply:
I had a feeling something like this existed but wasn't sure what it was called, thanks!